.gitlab-ci.yml

```
@@ -320,6 +320,27 @@ variables:
       AGENT_REPOSITORY: agent
       IMG_REGISTRIES: public
 
+# Same as on_deploy_a6_manual, except the job would not run
+# on RC pipelines, it would only run for the final release.
+.on_deploy_a6_manual_final:
+  - <<: *if_not_version_6
+    when: never
+  - <<: *if_not_deploy
+    when: never
+  - <<: *if_rc_tag_on_beta_repo_branch
```

I just thought about something: the 4 main modes of deploy pipelines we have are:
- final artifact -> stable (Docker jobs target agent)
- RC artifact -> beta (Docker jobs target agent)
- custom build artifact -> beta (Docker jobs target agent)
- nightly artifacts -> nightly (Docker jobs target agent-dev)

We don't want the `deploy_latest` jobs to be run on RCs, but I think we also never want a custom build to accidentally get promoted to latest, so maybe doing:
```suggestion
  - <<: *if_beta_repo_branch
```

(that rule doesn't exist yet) is better (and simpler)?

---

Good point, I'll do that. Thanks!